src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        Color[] values = Color.values();
+        int randomIndex = new Random().nextInt(values.length);
+        return values[randomIndex].name();
```

Comment:
Consider changing the return type of `getRandomColor` from `String` to `Color` to maintain consistency with the `Color` enum. This will allow you to work directly with the `Color` type instead of converting it to a `String`.